src/styles/blocks/testimonials.scss

```diff
@@ -0,0 +1,240 @@
+.testimonials {
+  padding-bottom: 120px;
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__subhead {
+    margin-bottom: 16px;
+  }
+
+  &__title {
+    font-weight: 600;
+    font-size: 32px;
+    line-height: 150%;
+    color: #253757;
+    margin-bottom: 48px;
+  }
+
+  &__card {
+    width: 100%;
+    background: white;
+    box-shadow: 0px 9px 18px rgba(37, 41, 49, 0.03);
+    border-radius: 16px;
+    text-align: center;
+    margin: 0 auto 24px;
+    z-index: 0;
+    transition: .3s;
+  }
+
+  &__card-container {
+    display: flex;
+    flex-wrap: nowrap;
+    flex-direction: column;
+    align-items: center;
+    padding: 76px 42px 32px;
+  }
+
+  &__img {
+    width: 110px;
+    height: 110px;
+    border-radius: 50%;
+    object-fit: cover;
+    object-position: center;
+    margin-bottom: 100px;
+    position: relative;
+
+    &::before {
+      position: absolute;
+      content: url(../images/testimonials/testimonials-img-rectangle-mobile.svg);
+      right: 3.2em;
+      bottom: 1.8em;
+      z-index: -1;
+    }
+
+    &::after {
+      position: absolute;
+      content: url(../images/testimonials/testimonials-img-rectangle-mobile.svg);
+      left: 3.1em;
+      top: 2.3em;
+      z-index: -1;
+    }
+  }
+
+  &__img--1 {
+    background: url(../images/testimonials/testimonial-ava-1.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
+  }
+
+  &__img--2 {
+    background: url(../images/testimonials/testimonial-ava-2.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
+  }
+
+  &__img--3 {
+    background: url(../images/testimonials/testimonial-ava-3.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
+  }
+
+  &__quote {
+    max-width: 196px;
+    color: var(--color-paragraph);
+    font-family: 'Open Sans';
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 150%;
+    margin-bottom: 16px;
+    position: relative;
+
+    &::before {
+      position: absolute;
+      content: url(../images/testimonials/quote-mark.svg);
+      left: 45%;
+      bottom: 100%;
+    }
+  }
+
+  &__cite {
+    color: #253757;
+    font-family: 'Open Sans';
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
```

Comment:
You can declare font-size in card selector. This font-size will apply to the text of the entire card.
Also good practice create some general selectors for text, for example: 
.text-size--1 {
  font-size: 24px;
}

.text-size--2 {
  font-size: 18px;
}